charts/system-upgrade-controller/system-upgrade-controller/100.0.1+up0.3.1/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/namespace: cattle-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/release-name: system-upgrade-controller
+apiVersion: v1
+appVersion: v0.8.0-rc.1
+description: General purpose controller to make system level updates to nodes
+home: https://github.com/rancher/system-charts/charts/system-upgrade-controller
+name: system-upgrade-controller
+sources:
+- https://github.com/rancher/system-charts/charts/system-upgrade-controller
```

Comment:
The home URL and the sources URL is invalid, 404 error is returned

Reply:
I didn't change those, they are the same as the previous version. Maybe @kinarashah can comment on that.

Reply:
@jiaqiluo What are the home and source URLs used for? This is sourced from upstream rancher/system-charts https://github.com/rancher/charts/blob/dev-v2.6/packages/system-upgrade-controller/package.yaml, and the released version 0.3.0 also has the same values https://github.com/rancher/charts/blob/dev-v2.6/charts/system-upgrade-controller/system-upgrade-controller/100.0.0%2Bup0.3.0/Chart.yaml.

Reply:
Original PR for more context: https://github.com/rancher/charts/pull/1422

Reply:
@kinarashah Those are general information of the chart, which are not related to any functionality of the chart. I think they are just out of date in the upstream chart, and we need to update them sooner or later.

Reply:
@jiaqiluo I see. Looks like we missed setting those correctly when sourcing from upstream for 0.3.0? 
I'm still confused on what the values should be, upstream chart by itself has the right path https://github.com/rancher/system-charts/blob/dev-v2.6/charts/rancher-k3s-upgrader/0.3.1/Chart.yaml#L5. Should these URLs point to `charts/system-upgrade-controller` or `system-charts/rancher-k3s-upgrader`?

Reply:
I think the url should be the `charts/system-upgrade-controller` one. see my comment below.